config/configuration-test-files/test-conf.json

```diff
@@ -8,5 +8,6 @@
   "service_token": "token",
   "supported_auth_types": ["x509", "oidc"],
   "supported_auth_methods": ["api-key", "x-api-token"],
+  "supported_service_types": ["ams", "web-api", "custom"],
```

Comment:
Here the supported service types are ["ams", "web-api", "custom"],
but in readme and swagger you refer to 
+   "supported_service_types": ["ams", "web-api"],

Reply:
updated readme and swagger